engine/graphics/src/null/graphics_null.cpp

```diff
@@ -61,6 +61,26 @@ namespace dmGraphics
 
     DM_REGISTER_GRAPHICS_ADAPTER(GraphicsAdapterNull, &g_null_adapter, NullIsSupported, NullRegisterFunctionTable, g_null_adapter_priority);
 
+    struct AssetContainerLock
```

Comment:
Maybe it's better to make it as part of dmsdk? IMO it's useful for extension developers.

Reply:
And if it will become a part of dmsdk - add similar implementation for dmSpinlock.